workspace/notebook/py_sb_raw_to_std.json

```diff
@@ -148,6 +172,108 @@
 				],
 				"execution_count": null
 			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"##### Initialize and define all functions required for Application Insight logging"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"# Initialize json processing results\r\n",
+					"processing_results = {\r\n",
+					"    \"processing_summary\":{\r\n",
+					"    \"total_tables_processed\": 0,\r\n",
+					"    \"successful_tables\": 0,\r\n",
+					"    \"failed_tables\": 0\r\n",
```

Comment:
Indentation could be improved here to make it clear these properties are a part of `processing_summary`